### PR TITLE
refactor: remove package and imports

### LINKED ISSUE #116: Remove old tui package and its imports

### PR DESCRIPTION
**Pull request by iRohitGaur**
### What is the change?
- removes the old tui calendar package
- removes the css import of tui calendar package
- addresses a typo in `navbar` component

Provide a small description of what did you change and provide the reference to the issue ticket.
Closes #116

### Is it bug?
No

### \*Dev Tested?

- [X] Yes
- [ ] No

#### Platforms

- [ ] Web

#### Browsers

- [ ] Chrome
- [ ] Safari
- [ ] Firefox

### Before / After Change Screenshots

> For visual or interaction changes. Can be video / screenshot.

### CODE REVIEW

**Review by isVivek99 — COMMENTED** [review]
have we made any change in the navbar file?

**Comment by iRohitGaur** [comment]
> have we made any change in the navbar file?

Yes. Addressed a typo in that file. Also mentioned the same in the PR description.

File: app/components/common/navbar/index.tsx [hunk 1]
```diff
@@ -144,19 +144,19 @@ const Navbar = () => {
...
+          {navbarElements.navbarSettings.map((navItem: NavItemType) => (
             <li
-              key={navIntem.id}
-              className={`${navIntem.visibleOnDesktop ? '' : 'md:hidden'}  ${
+              key={navItem.id}
+              className={`${navItem.visibleOnDesktop ? '' : 'md:hidden'}  ${
                 classShortHands.flexDesktopRow
               } items-center mb-0 md:mb-8 cursor-pointer group`}
-              onClick={() => toggle(navIntem.id)}
+              onClick={() => toggle(navItem.id)}
```
Comment (harshith-venkatesh) [inline app/components/common/navbar/index.tsx:153]: Good work in catching this typo🙂

File: package.json [hunk 2]
```diff
@@ -32,7 +32,6 @@
     "@remix-run/node": "^1.9.0",
     "@remix-run/react": "^1.7.4",
     "@remix-run/serve": "^1.7.4",
-    "@toast-ui/react-calendar": "^2.1.3",
```
Comment (harshith-venkatesh) [inline package.json:35]: 👍

**Review by sakshambhatt — APPROVED** [review]
LGTM :rocket:

**Review by isVivek99 — APPROVED** [review]

**Pull request merged**